scipy/spatial/transform/rotation.py

```diff
@@ -629,3 +629,31 @@ def inv(self):
         if self._single:
             quat = quat[0]
         return self.__class__(quat, normalized=True)
+
+    def __mul__(self, other):
+        """Returns the composition of the rotations.
+
+        If `p` and `q` are two rotations, then the composition of 'q followed
+        by p' is equivalent to `p * q`, which is the same as
```

Comment:
I suggest to rephrase it as: "In terms of DCM, the composition is expressed as ...". That is be slightly more clear, that we can consider the composition as dcm multiplication.